eng/pipelines/libraries/prepare-publish-for-tests.yml

```diff
@@ -0,0 +1,55 @@
+# Steps used to prepare the Artifacts Staging Directory with required files for
```

Comment:
Nit: maybe name this template, `prepare-publish-for-framework` or something like that? Cause this is not just for tests, this is also for the official build.